MediaBrowser.Providers/MediaInfo/FFProbeProvider.cs

```diff
@@ -62,8 +62,8 @@ public class FFProbeProvider : ICustomMetadataProvider<Episode>,
             NamingOptions namingOptions)
         {
             _logger = logger;
-            _audioResolver = new AudioResolver(localization, mediaEncoder, fileSystem, namingOptions);
-            _subtitleResolver = new SubtitleResolver(localization, mediaEncoder, fileSystem, namingOptions);
+            _audioResolver = new AudioResolver(logger, localization, mediaEncoder, fileSystem, namingOptions);
```

Comment:
It should be `ILoggerFactory` instead of `ILogger<FFProbeProvider>` like this https://github.com/jellyfin/jellyfin/blob/dc8fdb154af7d0b71ca15a551b2acd4d2fe3dbd1/Emby.Dlna/DlnaManager.cs#L51

Reply:
fixed.